sound/soc/sof/ipc4-topology.c

```diff
@@ -890,7 +890,7 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 	}
 
 	cfg_size = sizeof(struct sof_ipc4_base_module_cfg);
-	if (process->payload_with_output_fmt)
+	if (process->init_payload_format == INIT_PAYLOAD_WITH_OUTPUT_FMT)
```

Comment:
Should we use `if (process->init_payload_format & INIT_PAYLOAD_WITH_OUTPUT_FMT)`? We may have other init payload at the same time, right?

Reply:
One possible option is the reference format, but only two modules have the optional reference audio format. MUX and WHM, MUX will not be supported with process ops as it is snd_soc_dapm_mux type, and WHM will not be supported in linux side. so == is good.